src/components/Tabbar/index.js

```diff
@@ -83,7 +78,7 @@ const Tabbar = ({state, descriptors, navigation, position}) => {
             testID={options.tabBarTestID}
             onPress={onPress}
             style={S.singletab}>
-            <Animated.Text style={{opacity, ...S.singletabtext}}>{label}</Animated.Text>
+            <Text style={[S.singletabtext, isFocused ? {color: COLORS.white} : {}]}>{label}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style object inside the Text component can be optimized by moving the conditional logic outside of the inline style. This way, you avoid creating a new object every time the component re-renders.
```diff
- <Text style={[S.singletabtext, isFocused ? {color: COLORS.white} : {}]}>{label}</Text>
+ const focusedStyle = isFocused ? {color: COLORS.white} : {};
+ <Text style={[S.singletabtext, focusedStyle]}>{label}</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->